lib/automations/assign-milestone/pull-request-review-handler.js

```diff
@@ -16,53 +17,81 @@ module.exports = async ( context, octokit ) => {
 	const pullNumber = context.payload.pull_request.number;
 	const reviewState = context.payload.review.state;
 
-	debug(
-		`pullRequestReviewHandler: Pull Request number is [${ pullNumber }].`
-	);
-	debug( `pullRequestReviewHandler: Review state is [${ reviewState }].` );
+	debug( `assign-milestone: Pull Request number is [${ pullNumber }].` );
+	debug( `assign-milestone: Review state is [${ reviewState }].` );
 
 	// Check state
 	if ( reviewState !== 'approved' ) {
-		debug(
-			`pullRequestReviewHandler: Review state is not approved--bailing.`
-		);
+		debug( `assign-milestone: Review state is not approved--bailing.` );
 		return;
 	}
 
 	// Check current milestone
 	if ( context.payload.pull_request.milestone !== null ) {
 		debug(
-			`pullRequestReviewHandler: Pull request already has a milestone--bailing.`
+			`assign-milestone: Pull request already has a milestone--bailing.`
 		);
 		return;
 	}
 
-	// Get next milestone
-	const milestones = await octokit.issues.listMilestones( {
+	// Get version.
+	debug( 'assign-milestone: Fetching `package.json` contents' );
+
+	const {
+		data: { content, encoding },
+	} = await octokit.repos.getContents( {
 		...context.repo,
-		sort: 'due_on',
-		direction: 'asc',
+		path: 'package.json',
 	} );
 
-	if ( ! milestones.data || ! milestones.data[ 0 ] ) {
+	const { version } = JSON.parse(
+		Buffer.from( content, encoding ).toString()
+	);
+
+	let [ major, minor ] = version.split( '.' ).map( Number );
+
+	debug(
+		`assign-milestone: Current plugin version is ${ major }.${ minor }`
+	);
+
+	// Calculate next milestone
+	if ( minor === 9 ) {
+		major += 1;
+		minor = 0;
+	} else {
+		minor += 1;
+	}
```

Comment:
Should this be wrapped in a try/catch in case the contents of `package.json` is malformed, or there is no `version` property? It would allow for early abort with a more contextual error?